include/drivers/display.h

```diff
@@ -81,6 +81,16 @@ enum display_orientation {
 	DISPLAY_ORIENTATION_ROTATED_270,
 };
 
+/**
```

Comment:
split the API changes to another commit.

Reply:
@parthitce Modified!